propagation-validation-server/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I think this package.json should be similar to what we have in packages including the license, author etc. etc